source/components/status-label/status-label.less

```diff
@@ -0,0 +1,56 @@
+hx-status {
+  background-color: transparent;
+  border: 1px solid;
+  border-color: @gray-750;
+  border-radius: 1em; // 10px
+  color: @gray-750;
+  display: inline-block;
+  font-size: 0.625rem; // 10px
+  line-height: 1em;
+  max-width: 100%;
```

Comment:
This property probably isn't doing what you expect. This is saying that its max width should be 100% of the content's width (which is default in most cases). I think you can remove this property.

If you're trying to solve the issue of a status label overflowing its container, consider using...
```less
text-overflow: ellipsis;
overflow: hidden;
```

Reply:
yep I removed max-width. Still can't get ellipsis to work unless the element has an explicit width. I will leave out for now and see if the grid stretches its children once we get it in our app. Rather not have min, max, or explicit widths on this custom element. Rather just let its content determine its width for now.